website/extensions/mlc.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+//import CodeBlock from '@theme/CodeBlock'
+import Admonition from '@theme/Admonition'
+import { reactrLanguages, reactrLanguageSupport } from '@site/reactr-lang.json'
+
+import Link from '@docusaurus/Link'
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+
+/* Creates a new iteratable, ordered list of supported languages filtered by only the code blocks present in the component */
+export const getCodeBlockLangs = (children) =>
```

Comment:
Just to double check, does this need exporting?

Reply:
IIRC you need the `export` keyword to define things in MDX, but perhaps not JSX, I'll double-check!

Reply:
This seems to work fine so a bunch of these have been removed in the latest version! Thanks!